### ISSUE TITLE
Integrate with --capture and -v options of pytest

### ISSUE THREAD

**Issue by abusalimov** [eisensheng/pytest-catchlog#14]
There's a [pytest-logging](https://github.com/saltstack/pytest-logging) plugin by @s0undt3ch that provides a nice way to control logging through the default `-v` and `-s` (alias for `--capture=no`) options of pytest-core. The verbosity flag naturally configures the logging level, and the capturing option - whether to output live logs for all tests, or only capture them to show in case of a failure afterwards.

For instance, one can run `py.test -vv -s` to see live logs at the `INFO` level or above emitted by all tests.

It would be nice to have something like that in pytest-catchlog.

**Comment by eisensheng** [comment]
This appears to be somewhat related to #1 but the solution provided by pytest-logging for changing log verbosity is by far more appealing because of the simplicity of the options IMHO. It would be nice if @s0undt3ch could provide a PR to integrate this feature in pytest-catchlog.

The options `--logging-format` and `--logging-date-format` provided by pytest-logging are already covered by pytest-catchlog.

**Comment by s0undt3ch** [comment]
I will provide a PR, just haven't gotten the time.

**Comment by s0undt3ch** [comment]
My PR will obsolete my plugin and once you decide to rename catchlog to logging I will also remove my plugin from pypi...

**Comment by eisensheng** [comment]
Perfectly fine. :wink: My day work also keeps me busy during the week so I just try to catch up with discussions on work days and try to devote some time to other projects like this one here during the weekend. 

Regarding the renaming we have a discussion running on the pytest mailing list. Maybe it does makes sense after all to rename this Plugin to pytest-logging. My opinion changed in favour to the rename.

**Comment by s0undt3ch** [comment]
Once my PR is merged and catchlog releases a new version, can someone ping me so that I update the read me to redirect people to catchlog?

**Comment by Nikratio** [comment]
When this functionality is integrated, please provide an option for catchlog to preserve the current behavior (i.e., do not print any log messages to stderr/stdout while the test is running, only report them afterwards and only if the test has failed).

**Comment by The-Compiler** [comment]
@Nikratio you mean you want that behaviour even with `--capture=no`, which displays all other output immediately?

**Comment by Nikratio** [comment]
Yes.

But more importantly, when using `--capture=fd` and encountering a failed test, I don't want the log messages to appear twice (as captured output and as captured log messages), and I would like to keep the distinction between captured output and captured log messages (so using only output capture without capturing log messages is not a solution).

**Comment by twmr** [comment]
This issue can already be closed. The live-logging functionality has already been integrated into pytest-core.

**Comment by The-Compiler** [comment]
Indeed, thanks!
